src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -65,10 +65,7 @@ struct OperationParameters {
             methodDecoding.append(param)
         }
 
-        // Add a blank key,value in order for Stencil generates an empty dictionary for QueryParams and PathParams constructor
-        if query.required.isEmpty {
-            query.required.append(KeyValueViewModel(key: "", value: "\"\""))
-        }
```

Comment:
What's the rationale for removing this?

Reply:
Populate empty strings in query dictionary will result in the url query parameters. i.e. the url building logic does not skip when the key is "".  Moreover, putting "" in query array in generated code is not needed. The generated code will still compile.